ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge-no-common-path-group.cfg

```diff
@@ -214,7 +214,6 @@ application traffic recognition
       application CUSTOM-VOICE-APPLICATION
    !
    field-set ipv4 prefix CONTROL-PLANE-APP-DEST-PREFIXES
```

Comment:
So this setup will not work since we have no pathfinders available. Should we error out in this case instead of producing a non-working config?

Reply:
so the purpose of the PR is to make sure we don't configure the neighbor which was done erroneously until now. In a scenario with multiple pathfinders where at least one would have a common path-group we want to configure only the ones with common path-groups.

However I agree that if there is no common path-group with ALL configured pathfinders, we should probably error out, though not sure if it could be valid for HA site for instance. Will track this and ask engineering